test/client/query-subscribe.js

```diff
@@ -2,7 +2,7 @@ var expect = require('expect.js');
 var async = require('async');
 var util = require('../util');
 
-module.exports = function() {
+module.exports = function(makeSortedQuery) {
```

Comment:
Actually, `getSortedQuery` is really just the generic query method. We also need to call it for all cases where we don't need a sort. If the query language were SQL, then we'd call it for a simple filtering query. We should update all the tests that make queries to go through the single function so that any format could be implemented. Should probably just use the name `getQuery`